app/qml/editor/inputcheckbox.qml

```diff
@@ -68,7 +68,7 @@ Item {
       leftPadding: customStyle.fields.sideMargin
     }
 
-    SwitchComponent {
+    Switch {
```

Comment:
is this right? switch is qt native and switch component is our???

Reply:
@tomasMizera @sklencar

Reply:
Its our custom widget and should be the only usage of the component. 
It has been renamed in #1405 and later on it may bring a bit of confusion indeed.

Reply:
@sklencar can you please rename it and make PR ?